tests/artist-create.test.js

```diff
@@ -0,0 +1,37 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const getDb = require('../src/services/db');
+const app = require('../src/app');
+
+describe('create artist', () => {
+  let db;
+  beforeEach(async () => {
+    db = await getDb();
+  });
+
+  afterEach(async () => {
+    await db.query('DELETE FROM Artist');
+    //await db.close();
+    await db.end();
+  });
+
+  describe('/artist', () => {
+    describe('POST', () => {
+      it('creates a new artist in the database', async () => {
```

Comment:
Could a test be added to assert what would happen if the artist name or genre was left blank in the request?